test/functional/kernel/nested-loop/tests/nested-loop-MultiLambda-impl.hpp

```diff
@@ -172,7 +172,9 @@ struct MultiLambdaNestedLoopExec<DEVICE_DEPTH_2, POLICY_DATA> {
           RAJA::statement::For<1, typename camp::at<POLICY_DATA, camp::num<1>>::type,
             RAJA::statement::Lambda<0>
           >
-        >,
+        >
+      >,
+      RAJA::statement::DEVICE_KERNEL<
```

Comment:
what was the issue? was it essentially launching instructions for two kernels?

Reply:
I updated the description, but the two loop nests are dependent.